src/main/java/com/google/api/codegen/transformer/go/GoImportSectionTransformer.java

```diff
@@ -25,7 +27,13 @@
 
 public class GoImportSectionTransformer implements ImportSectionTransformer {
   @Override
-  public ImportSectionView generateImportSection(SurfaceTransformerContext context) {
+  public ImportSectionView generateFileHeaderImportSection(SurfaceTransformerContext context) {
+    return generateImportSection(context.getTypeTable().getImports());
```

Comment:
I think I can separate it into two interfaces, so only the Python one and the standard one needs to implement both.